packages/extension-api/src/extension-api.d.ts

```diff
@@ -4300,6 +4300,8 @@ declare module '@podman-desktop/api' {
     export function setValue(key: string, value: any, scope?: 'onboarding'): void;
   }
 
+  export type CliToolInstallationSource = 'userInstalled' | 'appInstalled';
```

Comment:
not sure to repeat the word 'installed' as the property name is installation source

is it like something more simple 'extension' or 'external' (as it may not be the user but the system or something else)

```typescript
 export type CliToolInstallationSource = 'extension' | 'external'
```

Reply:
updated